src/frontend/src/flows/authorize/index.ts

```diff
@@ -138,14 +138,12 @@ export const authnTemplateAuthorize = ({
  */
 export const authFlowAuthorize = async (
   connection: Connection
-): Promise<void> => {
+): Promise<never> => {
   const i18n = new I18n();
-  const container = document.getElementById("pageContent") as HTMLElement;
+  const dapps = shuffleArray(getDapps());
```

Comment:
You don't need to shuffle here, right? Only a single logo is needed.

Reply:
ah, that's why it wasn't being shuffled. Thanks!

Reply:
Clarified (inlined). Thanks!